src/bin/winuser/dllmain.c

```diff
@@ -34,7 +34,9 @@ DllMain(
     switch (Reason) {
 
     case DLL_PROCESS_ATTACH:
+#ifndef _MT
```

Comment:
Can you put a comment on this line (and in the other file) saying something like "Don't disable thread library calls with static CRT!"

Reply:
Done

Reply:
Just put the comment on the same line as the #ifdef please:
```
#ifndef _MT // Don't disable thread library calls with static CRT!
```